src/mcts/search.cc

```diff
@@ -108,9 +108,15 @@ void Search::SendUciInfo() REQUIRES(nodes_mutex_) {
   }
   common_info.hashfull =
       cache_->GetSize() * 1000LL / std::max(cache_->GetCapacity(), 1);
-  common_info.nps =
-      common_info.time ? (total_playouts_ * 1000 / common_info.time) : 0;
-  common_info.tb_hits = tb_hits_.load(std::memory_order_acquire);
```

Comment:
This `tb_hits` removal was probably unintentional?

Reply:
Indeed it was! Returned it back.